libs/langchain/langchain/llms/vertexai.py

```diff
@@ -194,9 +223,41 @@ async def _acall(
             The string generated by the model.
         """
         return await asyncio.wrap_future(
-            self._get_task_executor().submit(self._predict, prompt, stop)
+            self._get_task_executor().submit(self._call, prompt, stop)
         )
 
+
+class VertexAIModelGarden(_VertexAIBase, LLM):
+    """Large language models served from Vertex AI Model Garden."""
+
+    client: "PredictionServiceClient" = None  #: :meta private:
+    project: str
+    "A GCP project where endpoint is hosted."
+    endpoint_id: str
+    "A name of an endpoint where the model has been deployed."
+    allowed_model_args: Optional[List[str]] = None
+    """Allowed optional args to be passed to the model."""
+    prompt_arg: str = "prompt"
+    result_arg: str = "generated_text"
+
+    @root_validator()
+    def validate_environment(cls, values: Dict) -> Dict:
+        """Validate that the python package exists in environment."""
+        from google.cloud.aiplatform.gapic import PredictionServiceClient
```

Comment:
the try/except ImportError should be around this line

Reply:
Done